vcap/vcap/testing/capsule_loading.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+import mock
+
+from vcap import BaseCapsule
+from vcap.loading.capsule_loading import load_capsule
+
+
+def load_capsule_with_one_device(packaged_capsule_path: Path) -> BaseCapsule:
+    """
+    Load the capsule, but patch out the DeviceMapper so that it never returns
+    multiple devices.
+
+    Essentially, this disables capsules from loading backends onto multiple
+    devices, for example GPUs. This is for the purpose of speeding up test
+    setup and teardown.
+    """
+
+    def mock_init(self, filter_func):
+        # Don't call the superclass init so that self.filter_func isn't
+        # overridden
+        self.filter_func = lambda all_devices: [filter_func(all_devices)[0]]
+
+    with mock.patch('vcap.device_mapping.DeviceMapper.__init__',
```

Comment:
Could this be replaced with a __qualname__ call?

Reply:
I don't think so... I'm pretty sure mock.patch wants to have the full path (including module) to the function. 

__qualname__ won't return that.

As far as maintainability, this is a test, so I think it has low risk of breaking and going unnoticed.